BonusCalcApi.Tests/Dockerfile

```diff
@@ -33,8 +26,8 @@ RUN dotnet restore ./BonusCalcApi.Tests/BonusCalcApi.Tests.csproj
 COPY . .
 
 RUN dotnet build -c debug -o out BonusCalcApi.Tests/BonusCalcApi.Tests.csproj
-RUN dotnet build -c Release -o out BonusCalcApi/BonusCalcApi.csproj
-
-CMD dotnet-coverage collect 'dotnet test' -f xml  -o 'coverage.xml'
+RUN dotnet build -c debug -o out BonusCalcApi/BonusCalcApi.csproj

```

Comment:
I changed this to debug to try to get coverage reports, however that might have been failing because I was running the container on my Mac (so much for being platform independent!)